gerrit.py

```diff
@@ -120,7 +121,7 @@ def main():
     gerrit_git = GerritGit(git_dir='gerrit_git_dir',
```

Comment:
How about we break all of this code in the main() function out into a test? (Since that is basically what it is.)

Reply:
any resources on starting tests for Python?